src/main/scala/tech/cryptonomic/conseil/metadata/UnitTransformation.scala

```diff
@@ -7,30 +7,50 @@ import tech.cryptonomic.conseil.util.OptionUtil.when
 class UnitTransformation(overrides: MetadataOverridesConfiguration) {
 
   def overridePlatform(platform: Platform, path: PlatformPath): Option[Platform] = when(overrides.isVisible(path)) {
-    platform.copy(displayName = overrides
-      .platform(path)
-      .flatMap(_.displayName)
-      .getOrElse(platform.displayName))
+    val overridePlatform = overrides.platform(path)
```

Comment:
I'm wondering if chimney would help here(it is already in the project)?: https://scalalandio.github.io/chimney/

Reply:
yes, it should be in the dependencies, so it might be worthwhile to try it out and see if it reads better

Reply:
It is not possible right now without complicating the code. 
I proposed a ticket to Chimney: https://github.com/scalalandio/chimney/issues/106